logdevice/common/configuration/nodes/ShardStateTracker.cpp

```diff
@@ -19,16 +19,20 @@ namespace {
 ShardState::Update
 transitionToNonIntermediaryState(const ShardState& shard_state) {
   ShardState::Update update{};
-  update.transition = StorageStateTransition::OVERRIDE_STATE;
-  update.conditions = Condition::FORCE;
-
-  ShardState::Update::StateOverride state_override{};
-  state_override.storage_state =
-      toNonIntermediaryState(shard_state.storage_state);
-  state_override.metadata_state =
-      toNonIntermediaryState(shard_state.metadata_state);
-  // TODO: handle UNRECOVERABLE flag
-  update.state_override = std::move(state_override);
+
+  if (shard_state.storage_state == StorageState::NONE_TO_RO) {
+    update.transition = StorageStateTransition::COMMIT_READ_ENABLED;
+    update.conditions = Condition::COPYSET_CONFIRMATION;
+  }
+  else if (shard_state.storage_state == StorageState::RW_TO_RO) {
```

Comment:
Can you run `clang-format` on the changes? :)

Reply:
done